### PR TITLE
Add tooltips to buttons in graphing calculator

### PR DESCRIPTION
**Pull request by EriWong**
### Description of the changes:
- Added tooltips to buttons in graphing calculator

### How changes were validated:
manually

### CODE REVIEW

File: src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml [hunk 1]
```diff
@@ -151,6 +151,7 @@
                             </Grid.RowDefinitions>
 
                             <ToggleButton x:Name="EquationButton"
+                                          x:Uid="equationAnalysisBack"
```
Comment (sanderl) [inline src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml]: This is showing hide instead of back

**Comment by sanderl** [comment]
I believe there's a small state icon too that just got checked in today.

File: src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml (hunk 1 above)
Comment (EriWong) [reply src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml]: good catch.  should be fixed now.

**Comment by EriWong** [comment]
> I believe there's a small state icon too that just got checked in today.

Rudy handled tooltips in his commit for small state already.

**Pull request merged**